File: Serveur/src/main/java/com/epul/metier/Personnage_.java
package main.java.com.epul.metier;

import javax.annotation.Generated;
import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;

@Generated(value="Dali", date="2016-10-25T08:43:06.892+0200")
@StaticMetamodel(Personnage.class)
public class Personnage_ {
	public static volatile SingularAttribute<Personnage, PersonnageId> id;
	public static volatile SingularAttribute<Personnage, Acteur> acteur;
	public static volatile SingularAttribute<Personnage, Film> film;
	public static volatile SingularAttribute<Personnage, String> nomPers;
}

File: Client/app/personnage/personnage.ts
import { Film } from '../film/film';
import { Acteur } from '../acteur/acteur';

export class Personnage {
    id: PersonnageId;
    film: Film;
    acteur: Acteur;
    nomPers: string;
}

export class PersonnageId {
    noFilm: number;
    noAct: number;

    constructor(noFilm: number, noAct: number) {
        this.noFilm = noFilm;
        this.noAct = noAct;
    }
}